### PR TITLE
Disable error popup and relocate in game performance metrics

### LINKED ISSUE #1116: Temporary UI fixes
1. Disable the error/reconnect popup until the issues are fixed.
2. Relocate the game stats bar (fps, ping, and hashes) to the top of the screen.

### PR DESCRIPTION
**Pull request by tkz00**
Closes #1116

## Motivation

The position of the in-game metrics was being covered in iOS by a system bar that appears in the bottom of the screen, so it had to be moved to the top. The version error popup was disrupting the game flow and doesn't currently make the verification correctly, it always appeared even if the game was playable, and the reconnect functionality isn't working as intended either so the popup had to be disabled temporarily.

## Summary of changes

Moved the game metric element to the top of the battle UI, also had to vertically mirror the metrics container image so it looked correctly on top of the screen, and disabled the call to the coroutine that checks for ongoing player games as well as the logic that makes the version error popup appear, both of them in the `LobbyConnection.cs` script.

## How has this been tested?

- For the in-game metrics display enter a game and in the battle scene you should see the fps, ping ms, and version hashes on top of the screen instead of the bottom.

- For the popups that don't have to appear enter a game and get out both by force quitting the game (stop playing in Unity or close the application in mobile) and check when you enter to the lobbies scene again the reconnect popup doesn't appear. Also check the version match error popup doesn't appear in the lobbies scene when you enter it or try to create a lobby.

## Test additions / changes

No tests were added or changed

## Checklist
- [x] I have tested the changes locally.
- [x] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

File: client/Assets/Scenes/Battle.unity
Comment (BertovDev) [inline client/Assets/Scenes/Battle.unity:1]: Changes should be made in `CameraUI` prefab and not the scene. 
You can aplly all the changes of the prefab in the scene and then checkout the changes with:

`git checkout main -- client/Assets/Scenes/Battle.unity` to restore the Battle Scene

File: client/Assets/Scripts/LobbyConnection.cs [hunk 1]
```diff
@@ -184,10 +184,10 @@ public void ConnectToLobby(string matchmaking_id)
...
     private void ValidateVersionHashes()
     {
-        if (serverHash.Trim() != GitInfo.GetGitHash().Trim())
-        {
-            Errors.Instance.HandleVersionHashesError(versionHashesTitle, versionHashesDescription);
-        }
+        // if (serverHash.Trim() != GitInfo.GetGitHash().Trim())
+        // {
+        //     Errors.Instance.HandleVersionHashesError(versionHashesTitle, versionHashesDescription);
+        // }
```
Comment (BertovDev) [inline client/Assets/Scripts/LobbyConnection.cs:190]: We want to keep this code if it is not been use? 

What do you think?, let me know.

**Review by BertovDev — CHANGES_REQUESTED** [review]

File: client/Assets/Scenes/Battle.unity
Comment (tkz00) [reply client/Assets/Scenes/Battle.unity:1]: Right! I'll let you know when I make the change.

Comment (tkz00) [reply client/Assets/Scenes/Battle.unity:1]: All ready!

File: client/Assets/Scripts/LobbyConnection.cs (hunk 1 above)
Comment (BertovDev) [reply client/Assets/Scripts/LobbyConnection.cs:189]: Nice! seems correct

**Review by BertovDev — APPROVED** [review]
LGTM

**Review by ncontinanza — APPROVED** [review]

**Pull request merged**